docs/v2-strategies/executors/orchestrator.md

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Move the contents of this file to a section in `/v2-strategies/executors/index.md`